Description: With carbon stripped from the source tree, we need to strip
 the carbon dir from the clean targets
Author: Paul Gevers <elbrus@debian.org>

Index: lazarus/lcl/Makefile.fpc
===================================================================
--- lazarus.orig/lcl/Makefile.fpc
+++ lazarus/lcl/Makefile.fpc
@@ -161,7 +161,6 @@ all: lclbase compiled intf
 cleanlaz: cleanall
 cleanall: clean
 	$(MAKE) -C interfaces clean
-	$(MAKE) -C interfaces/carbon clean
 	$(MAKE) -C interfaces/cocoa clean
 	$(MAKE) -C interfaces/customdrawn clean
 	$(MAKE) -C interfaces/fpgui clean
Index: lazarus/lcl/interfaces/Makefile.fpc
===================================================================
--- lazarus.orig/lcl/interfaces/Makefile.fpc
+++ lazarus/lcl/interfaces/Makefile.fpc
@@ -2,7 +2,7 @@
 main=lcl
 
 [target]
-dirs=gtk gtk2 gtk3 win32 wince qt qt5 carbon fpgui nogui cocoa customdrawn mui
+dirs=gtk gtk2 gtk3 win32 wince qt qt5 fpgui nogui cocoa customdrawn mui
 
 [default]
 dir=$(LCL_PLATFORM)
